api/turing/testdata/cluster/servicebuilder/router_configmap_traffic_splitting.yml

```diff
@@ -122,5 +158,7 @@ strategy:
         values:
         - region-b
       route_id: traffic-split-1
+    - conditions: null
```

Comment:
It may be better to use `[]` here instead of `null` to avoid any special handling.